emoji/src/main/java/com/vanniktech/emoji/EmojiArrayAdapter.java

```diff
@@ -55,6 +62,23 @@ public View getView(final int position, final View convertView, @NonNull final V
       }
     });
 
+    if (!EmojiManager.getInstance().findSkinTonedEmojis(checkNotNull(getItem(position), EMOJI_NULL_ERROR)).isEmpty()) {
```

Comment:
you can extract the variable `checkNotNull(getItem(position), EMOJI_NULL_ERROR)` here